plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/struct/DBStructUtils.java

```diff
@@ -122,10 +122,17 @@ public static String generateObjectDDL(@NotNull DBRProgressMonitor monitor, @Not
         return SQLUtils.generateCommentLine(object.getDataSource(), "Can't generate DDL: object editor not found for " + object.getClass().getName());
     }
 
-    public static String getTableDDL(@NotNull DBRProgressMonitor monitor, @NotNull DBSEntity table, Map<String, Object> options, boolean addComments) throws DBException {
+    public static String getTableDDL(@NotNull DBRProgressMonitor monitor, @NotNull DBSEntity table, Map<String, Object> options,
```

Comment:
Per our code style, when wrapping method parameters, each parameter must be on its own line:
```
void method(
    String a,
    int b
) throws X {
    ...
}
```